shimmer/learn/shimmer-questions.json

```
@@ -0,0 +1,83 @@
+[
+    {
+      "questionText": "The staging and innovation network of IOTA is called ...?",
+      "answerOptions": [
+        { "answerText": "Assembly", "isCorrect": false },
+        { "answerText": "Soon Network", "isCorrect": false },
+        { "answerText": "IOTA 2.0", "isCorrect": false },
+        { "answerText": "Shimmer", "isCorrect": true }
+      ]
+    },
+    {
+      "questionText": "What's the beginning of a Shimmer Testnet address?",
```

AFAIK it's iether the Shimmer Beta network or just Testent right? As Shimmer is now live, I'd say we stick to just Testnet.

```suggestion
      "questionText": "What's the beginning of a Testnet address?",
```